lib/Dialect/AIEX/IR/AIEXDialect.cpp

```diff
@@ -203,7 +204,8 @@ LogicalResult AIEX::NpuDmaMemcpyNdOp::verify() {
     return emitOpError(msg.str());
   }
 
-  if (strides[0] != 1)
+  // FIXME: skip inner dim
+  if (raw_strides[0] != 1)
```

Comment:
I'm temporarily asserting `strides[0] == 1` and ignoring it.

Reply:
No, this is temporary to catch places I missed updating implicit stride 1 to explicit stride 1. I will fix